device/iot-device-client/src/test/java/com/microsoft/azure/sdk/iot/device/edge/DirectMethodResponseTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (c) Microsoft. All rights reserved.
+ *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+
+package com.microsoft.azure.sdk.iot.device.edge;
+
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonNull;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static junit.framework.TestCase.assertEquals;
+
+public class DirectMethodResponseTest
+{
+    // Tests_SRS_DIRECTMETHODRESULT_34_001: [This function shall return the saved status.]
+    // Tests_SRS_DIRECTMETHODRESULT_34_002: [This function shall return the saved status.]
+    // Tests_SRS_DIRECTMETHODRESULT_34_003: [This constructor shall retrieve the payload and status from the provided json.]
```

Comment:
We can get rid of these lines, right @timtay-microsoft ?

Reply:
Yep